crates/pallets/starknet/src/tests.rs

```diff
@@ -494,86 +491,117 @@ fn given_erc20_transfer_when_invoke_then_it_works() {
         System::set_block_number(0);
         run_to_block(1);
         let origin = RuntimeOrigin::none();
-        let (sender_account, _, _) = account_helper(TEST_ACCOUNT_SALT);
-        // Declare ERC20 contract
-        declare_erc20(origin.clone(), sender_account);
+        let sender_account = <[u8; 32]>::from_hex("000000000000000000000000000000000000000000000000000000000000000F").unwrap();
+        // ERC20 is already declared for the fees.
         // Deploy ERC20 contract
-        deploy_erc20(origin.clone(), sender_account);
+        let deploy_transaction = Transaction {
```

Comment:
why not use `transaction_from_json` ?

Reply:
Easier to assert the event and easier to modify. Imo tx from json should be used for real txs that we take from mainnet/testnet

Reply:
Makes sense